src/test/resources/kafka/project-created.json

```diff
@@ -1,16 +1,16 @@
 {
   "@context": "https://bluebrain.github.io/nexus/contexts/admin.json",
   "@type": "ProjectCreated",
-  "uuid": "a5c736f2-0ace-48f4-a7a7-56a15123d0b3",
-  "organization": "4cd2c88b-ed73-4fd7-8afb-315032239a56",
-  "label": "project label",
+  "_uuid": "a5c736f2-0ace-48f4-a7a7-56a15123d0b3",
+  "_organization": "4cd2c88b-ed73-4fd7-8afb-315032239a56",
+  "_label": "project label",
   "description": "description",
   "base": "https://nexus.example.com/base",
   "apiMappings": {
```

Comment:
The `apiMappings` seems to be encoded differently for events than in the API response. I guess it should be:

```json
{
   "apiMappings" : {
      "prefix": "nxv",
      "namespace": "https://bluebrain.github.io/nexus/vocabulary/"
   }
}
```